hive-metastore/src/main/java/org/apache/iceberg/hive/HiveCatalog.java

```diff
@@ -74,9 +74,18 @@
   public HiveCatalog() {
   }
 
+  /**
+   * Hive Catalog constructor.
+   *
+   * @deprecated please use the no-arg constructor, setConf and initialize to construct the catalog. Will be removed in
+   * v0.12.0
```

Comment:
0.12.0 is probably the next release. Can you change this to 0.13.0?

Also, now that 0.11.0 is out, can we remove the methods to remove in 0.12.0?

Reply:
done, also removed the `HadoopCatalog` constructors